src/radical/pilot/agent/scheduler/archive/continuous_summit.py

```diff
@@ -128,15 +125,14 @@ def initialize_child(self):
         # during agent startup.  We dig them out of the config at this point.
         #
         # NOTE: this information is insufficient for the torus scheduler!
-        self._pilot_id = self._cfg['pilot_id']
-        self._lrms_info = self._cfg['lrms_info']
-        self._lrms_lm_info = self._cfg['lrms_info']['lm_info']
-        self._lrms_node_list = self._cfg['lrms_info']['node_list']
+        self._pilot_id              = self._cfg['pilot_id']
```

Comment:
DIdn't we say that the `ContinuousSummit` scheduler is not needed? Am I forgetting something?